src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -659,7 +680,7 @@ private function fillNextRows(array $rows, int $line): array
     {
         $unmergedRows = [];
         foreach ($rows[$line] as $column => $cell) {
-            if (null !== $cell && !$cell instanceof TableCell && !is_scalar($cell) && !(\is_object($cell) && method_exists($cell, '__toString'))) {
+            if (null !== $cell && !$cell instanceof TableCell && !\is_scalar($cell) && !(\is_object($cell) && method_exists($cell, '__toString'))) {
```

Comment:
this change should be reverted as (it's useful for some functions only, `is_scalar` is not in the list).